docs/src/troubleshooting.md

```diff
@@ -81,12 +81,28 @@ there are a number of things you can try to fix such errors:
   side by side.
 * Instead of installing Trixi and conflicting packages in the same (default) environment,
   consider creating new environments/projects and install only packages required for
-  the specific tasks, as explained in the 
+  the specific tasks, as explained in the
   [official `Pkg` documentation](https://julialang.github.io/Pkg.jl/v1/environments/).
   For example, if you use Trixi for a research project (Bachelor/Master thesis or a paper),
-  you should create a new Julia project/environment for that research and add Trixi as a 
+  you should create a new Julia project/environment for that research and add Trixi as a
   dependency. If you track all your code and the `Project.toml`, `Manifest.toml` files
   (generated by `Pkg`) in a version control system such as `git`, you can make your research
   easily reproducible (if you also record the version of Julia you are using and leave some
   comments for others who do not know what you are trying to do, including your future self 😉).
 
+
+## [I see lots of questions marks and weird symbols in the output of Trixi](@id font-issues)
```

Comment:
```suggestion
## [There are many questions marks and weird symbols in the output of Trixi](@id font-issues)
```
We did not use first person perspective in the other item - but this is really a minor change and can also be ignored.